dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -94,11 +153,13 @@ public boolean equals(@Nullable Object obj) {
     }
 
     @Override
+    @Pure
     public int hashCode() {
         return Objects.hash(constructor, arguments);
     }
 
     @Override
+    @Pure
```

Comment:
Same here, isn't this only side-effect-free, but not deterministic (therefore, not pure)?

Reply:
Thanks!